ani-cli

```diff
@@ -248,7 +249,12 @@ open_episode () {
 			s/^${selection_id}\t[0-9]+/${selection_id}\t$((episode+1))/
 		" "$logfile" > "${logfile}.new" && mv "${logfile}.new" "$logfile"
 
-		setsid -f $player_fn --http-header-fields="Referer: $embedded_video_url" "$video_url" >/dev/null 2>&1
+		#In order for playback history to be recorded MPV needs to be used
+		if [ $player_fn == "mpv" ]; then
```

Comment:
we are using `http-header-fields` option which makes this script only work with `mpv`. So no need to do this check for now

Reply:
Should we leave it in for future use? I don't think it makes much of a difference either way.